index.js

```diff
@@ -21,6 +21,7 @@ var isWeakSet      = require('lodash/isWeakSet');
 
 var NEWLINE_REGEXP = /\n/ig;
 var SYMBOL_REGEXP = /^Symbol\((.*)\)(.*)$/;
+var plugins = [];
```

Comment:
Shouldn't need to have a global `plugins` if you just pass around `opts` which has `plugins` defined on it.